src/app/dragdrop/dragdrop.component.html

```diff
@@ -0,0 +1,49 @@
+Formatted HTML:
+<div class="header">
```

Comment:
think i'm seeing this header in every component page.
why not include it in the base template instead?